### PR TITLE
Removed doc review from new files and added deprecated command template

### PR DESCRIPTION
**Pull request by bziser**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-420

## Description
* Fixed an issue where the **doc-review** command reviewed new files.
* Added the `Command XXX is deprecated. Use XXX instead.` release notes templates to **doc-review** command.

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -595,11 +595,13 @@ class FileType(str, Enum):
 DEPRECATED_DESC_REGEX = r"Deprecated\.\s*(.*?Use .*? instead\.*?)"
 DEPRECATED_NO_REPLACE_DESC_REGEX = r"Deprecated\.\s*(.*?No available replacement\.*?)"
 PACK_NAME_DEPRECATED_REGEX = r".* \(Deprecated\)"
+DEPRECATED_COMMAND_REGEX = r"Command \*\*\*.*?\*\*\* is deprecated. Use .*? instead."
```
Comment (dorschw) [inline demisto_sdk/commands/common/constants.py:598]: What happens when there's no alternative?

Comment (bziser) [reply demisto_sdk/commands/common/constants.py:598]: It fails like it was before in the bug

**Review by dorschw — APPROVED** [review]

**Review by dorschw — CHANGES_REQUESTED** [review]
Let's revert the `added` part

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 * Added support for **xdrctemplate** to content graph.
 * Fixed an issue where local copies of the newly-introduced `DemistoClassApiModule.py` were validated.
 * Added new release notes templates for the addition and modification of playbooks, layouts and types in the **doc-review** command.
+* Fixed an issue where the **doc-review** command reviewed new files.
```
Comment (dorschw) [inline CHANGELOG.md:11]:
```suggestion
```

File: demisto_sdk/commands/common/constants.py [hunk 3]
```diff
@@ -595,11 +595,13 @@ class FileType(str, Enum):
...
 
 DEPRECATED_REGEXES: List[str] = [
     DEPRECATED_DESC_REGEX,
     DEPRECATED_NO_REPLACE_DESC_REGEX,
-    PACK_NAME_DEPRECATED_REGEX
+    PACK_NAME_DEPRECATED_REGEX,
+    DEPRECATED_COMMAND_REGEX
```
Comment (dorschw) [inline demisto_sdk/commands/common/constants.py:604]:
```suggestion
    DEPRECATED_COMMAND_REGEX,
```

File: demisto_sdk/commands/doc_reviewer/tests/rn_checker_test.py [hunk 4]
```diff
@@ -26,7 +26,14 @@
     (['Created a new layout for MITRE Att&ck.'], True),
     (['Playbook now supports IPs as well as Emails.'], True),
     (['Created a new playbook for CVE-XXXX-XXXX.'], True),
-    (['Updated the IP type regex.'], True)
+    (['Updated the IP type regex.'], True),
+    (['##### New: script',
+      '***command-one***',
+      '***command-two***'], True),
```
Comment (dorschw) [inline demisto_sdk/commands/doc_reviewer/tests/rn_checker_test.py:32]:
```suggestion
    (['##### New: script',
      '***command-one***',
      '***command-two***'], True),
          (['##### New: script',
      'some string that doesn't match a pattern',
      '🐼'], True),
```

File: CHANGELOG.md (hunk 2 above)
Comment (dorschw) [inline CHANGELOG.md:11]:
```suggestion
* Fixed an issue where the **doc-review** command failed on descriptions of new content items.
```

**Review by dorschw — APPROVED** [review]

**Pull request merged**